charts/bpa/templates/_helpers.tpl

```diff
@@ -93,7 +93,7 @@ generate hosts if not overriden
 generate ledger browser url
 */}}
 {{- define "bpa.ledgerBrowser" -}}
-{{- $ledgerBrowser := dict "bosch-test" "https://indy-test.bosch-digital.de" "idu" "https://explorer.idu.network" -}}
+{{- $ledgerBrowser := dict "bosch-test" "https://indy-test.bosch-digital.de" "idu" "https://explorer.idu.network" "bcovrin-test" "http://test.bcovrin.vonx.io" -}}
```

Comment:
Please check the other places where we have such a  "switch case" ( just search for "bosch-test") .
If I remember it correctly there some similar fallunterscheidungen